src/components/CreateLists.js

```diff
@@ -0,0 +1,24 @@
+import {getItem, OPENED_DOCUMENTS} from "../utils/storage.js"
+
+export const createList = (currentList) => {
+    const openedDocuments = getItem(OPENED_DOCUMENTS,[])
+    return currentList.map(({title,documents,id})=>{
+        const isOpen = openedDocuments.includes(String(id));
+        return`<ul data-id="${id}">
+        <div class="document__title">
+            <span>
+                <button class="toggle__button">></button>
```

Comment:
```suggestion
                <button class="toggle__button">&gt;</button>
```

HTML에서는 마크업 문제가 있을 수 있으니 `&gt;` `&lt;` 를 이용하길 권장합니다.